lale/lib/rasl/min_max_scaler.py

```diff
@@ -41,26 +41,59 @@ def __init__(self, feature_range=(0, 1), *, copy=True, clip=False):
             raise ValueError("`copy=False` is not supported by this implementation")
         if clip:
             raise ValueError("`clip=True` is not supported by this implementation")
-        self._first_fit = True
+        self.n_samples_seen_ = 0
+        self._hyperparams = None
```

Comment:
Instead of assigning feature_range to an attribute, I would suggest putting it in the self._hyperparams dictionary, alongside copy and clip.

Reply:
Thanks for the feedback. It is changed.